front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -301,35 +312,50 @@ export default function AssistantBuilder({
   );
 
   const onAssistantSave = async () => {
-    setDisableUnsavedChangesPrompt(true);
-    setIsSavingOrDeleting(true);
-    const res = await submitAssistantBuilderForm({
-      owner,
-      builderState,
-      agentConfigurationId,
-      slackData: {
-        selectedSlackChannels: selectedSlackChannels || [],
-        slackChannelsLinkedWithAgent,
-      },
-    });
-
-    if (res.isErr()) {
-      setIsSavingOrDeleting(false);
-      sendNotification({
-        title: "Error saving Assistant",
-        description: res.error.message,
-        type: "error",
-      });
+    if (!submitEnabled) {
+      // Redirect to the right screen if there are errors.
+      if (instructionsError) {
+        setScreen("instructions");
+      } else if (actionsErrors.length) {
+        setScreen("actions");
+      } else if (assistantHandleError || descriptionError) {
+        setScreen("naming");
+      } else {
+        console.log(
```

Comment:
What's the goal of this log? Is it for devs or for the users? 
If it's for devs, we might want to get rid of it and replace by a comment if we think it's necessary, and if it's for users better use a notification??

Reply:
It's for the dev, i wanted to log to datadog actually but couldn't find how.
I'll replace by a comment.